lib/copyResources.js

```diff
@@ -21,7 +21,7 @@ var unlinkIfExists = function (filename) {
     });
 };
 
-var IOS_ICON_SIZES = ['20', '40', '57', '72', '76', '114', '120', '144', '152', '167', '180', '1024'];
+var IOS_ICON_SIZES = ['20', '40', '60', '80', '57', '72', '76', '114', '120', '144', '152', '167', '180', '1024'];
```

Comment:
80 comes after 76.
It is better to keep track that way

Reply:
and 60 should be after 52

Reply:
Got it.Updated